test/torchaudio_unittest/prototype/functional/functional_test_impl.py

```diff
@@ -107,3 +108,77 @@ def test_add_noise_length_check(self):
 
         with self.assertRaisesRegex(ValueError, "Length dimensions"):
             F.add_noise(waveform, noise, lengths, snr)
+
+    @skipIfNoModule("pyroomacoustics")
+    @parameterized.expand([(2, 1), (3, 4)])
+    def test_simulate_rir_ism_single_band(self, D, channel):
+        """Test simulate_rir_ism when absorption coefficients are identical for all walls."""
```

Comment:
I do not see the test logic checks the identity described in the docstring.
It is checking against pyroomacoustic, but the test logic does not check identity for walls.

Can you elaborate?

Reply:
Yes. if `e_absorption` is just a floating-point value, then it means it is identical to all walls. That's why the room in pyroonacoustics is constructed with `materials=pra.Material(e_absorption),`.
The difference of this test and the one below is if `e_absorption` is a 2D Tensor, the multi-band logic in https://github.com/pytorch/audio/pull/2644/files#diff-bfbe08000c5dc05901af3bad9503693a99068426351748925a322990daed0cb3R217-R222 is applied. We need a separate test for that.